qcodes/instrument_drivers/QuantumDesign/DynaCoolPPMS/DynaCool.py

```diff
@@ -91,30 +93,43 @@ def __init__(self, name: str,
 
         self.add_parameter('field',
                            label='Field strength',
-                           unit='A/m',
+                           unit='T',
+                           get_cmd=self._deprecated_field_getter,
+                           snapshot_value=False)
+
+        self.add_parameter('field_measured',
+                           label='Field',
+                           unit='T',
                            get_cmd=self._present_field_getter)
 
+        self.add_parameter('field_target',
+                           label='Field target',
+                           unit='T',
+                           get_cmd=None,
+                           set_cmd=None,
+                           vals=vals.Numbers(-14, 14))
+
         self.add_parameter('field_setpoint',
                            label='Field setpoint',
-                           unit='A/m',
+                           unit='T',
```

Comment:
actually since the units are backwards INcompatible, perhaps, we can safely get rid of the "deprecated" parameters now? or not?

Reply:
I'd say we keep them for a little while.